ui/src/index.css

```diff
@@ -11,3 +16,7 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.MuiCard-root {
+  overflow: visible !important;
```

Comment:
Is it possible to not use `!important`? Same reason as above, to avoid breaking other parts of the styling.

Reply:
It was the only way I got to fix the error with the [album card panel being hidden.](https://github.com/navidrome/navidrome/pull/941#issuecomment-812303419). I'll take some time to properly understand MaterialUI stylings and will fix this